hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -145,18 +145,6 @@ public void initialize(URI name, Configuration conf) throws IOException {
           .build();
       LOG.trace("Ozone URI for ozfs initialization is {}", uri);
 
-      //isolated is the default for ozonefs-lib-legacy which includes the
-      // /ozonefs.txt, otherwise the default is false. It could be overridden.
-      boolean defaultValue =
-          BasicOzoneFileSystem.class.getClassLoader()
-              .getResource("ozonefs.txt")
-              != null;
-
-      //Use string here instead of the constant as constant may not be available
-      //on the classpath of a hadoop 2.7
-      boolean isolatedClassloader =
-          conf.getBoolean("ozone.fs.isolated-classloader", defaultValue);
```

Comment:
Can you remove OzoneConfigKeys#OZONE_FS_ISOLATED_CLASSLOADER as well?

Reply:
done.